src/daemon/ca/events.rs

```diff
@@ -376,31 +376,58 @@ impl fmt::Display for RoaUpdates {
 pub struct ChildCertificateUpdates {
     issued: Vec<IssuedCert>,
     removed: Vec<KeyIdentifier>,
+    suspended: Vec<SuspendedCert>,
 }
 
 impl ChildCertificateUpdates {
-    pub fn new(issued: Vec<IssuedCert>, removed: Vec<KeyIdentifier>) -> Self {
-        ChildCertificateUpdates { issued, removed }
+    pub fn new(issued: Vec<IssuedCert>, removed: Vec<KeyIdentifier>, suspended: Vec<IssuedCert>) -> Self {
+        ChildCertificateUpdates {
+            issued,
+            removed,
+            suspended,
+        }
     }
 
     pub fn is_empty(&self) -> bool {
         self.issued.is_empty() && self.removed.is_empty()
     }
+
+    /// Add an issued certificate to the current set of issued certificates.
```

Comment:
Doesn't it say above that the child is suspended which means its certificates are unpublished and revoked, i.e. is it right to talk about suspension and unsuspension of the _certificate_ here?